Tests/conf.json

```diff
@@ -1145,6 +1145,14 @@
             "integrations": "AWS Sagemaker"
 
         },
+        {
+            "playbookID": "C2sec-Test",
+            "integrations": "C2sec"
+
+        },
+        {
+            "playbookID": "C2SEC-Domain_Scan-Test"
```

Comment:
it doesn't really test any thing, you can remove the line the this test playbook

Reply:
removed